int16[2] position_cdeg
int16[2] velocity_cgeg
int16[2] current_100uA